app/src/main/java/com/oracle/javafx/scenebuilder/app/about/AboutWindowController.java

```diff
@@ -195,10 +202,55 @@ private StringBuilder getJavaFXParagraph() {
     private StringBuilder getJavaParagraph() {
         StringBuilder sb = new StringBuilder("Java\n"); //NOI18N
         sb.append(System.getProperty("java.runtime.version")).append(", ") //NOI18N
-                .append(System.getProperty("java.vendor")).append("\n\n"); //NOI18N
+                .append(System.getProperty("java.vendor")) // NOI18N
+                .append("\n\n"); 
+        
         return sb;
     }
     
+    private StringBuilder getJavaLibraryPathParagraph() {
+        StringBuilder sb = new StringBuilder("Java Library Path(s):\n"); //NOI18N
+        
+        String libPaths = System.getProperty("java.library.path");       //NOI18N
+        List<String> invalidPaths = new ArrayList<>();
+        String separator = getPathSeparator();
+        for (String libPath : libPaths.split(separator)) {
+            try {
+                Path absolutePath = Paths.get(libPath).normalize().toAbsolutePath();
+                if (Files.exists(absolutePath)) {                    
```

Comment:
Remove extraneous whitespace

Reply:
Which whitespace? The empty line or the indentation?

Reply:
Its removed now.